src/App.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {
+  Routes, Route, Link,
+} from 'react-router-dom';
+import Book from './Components/Books';
+import AddBook from './Components/Form';
+
+const NavBar = () => (
+  <nav className="NavBar">
+    <h2 className="NavTitle">Booklist</h2>
+    <Link to="/">Books</Link>
+    <Link to="/categories">Categories</Link>
+  </nav>
+);
+
+const Books = () => (
+  <div className="HomePage">
+    <ul className="Booklist">
+      <Book />
+      <Book />
+    </ul>
```

Comment:
- [ ] The book component should have a title and author props passed.
```js
<Book  title="book-title" author="book-author" />
```

- [OPTIONAL] You can make a small array of objects which hold a few books loop through them and render the book component with the book props passed.

Example:
```js
const tempBooks = [
         {id: 1, title: "Book 1", author: "author 1"},
         {id: 2, title: "Book 2", author: "author 2"}
];

<ul className="Booklist">
{tempBooks.map((book) => (
   <div key={book.id}>
       <Book  title={book.title} author={book.author} />
    </div>
)}
</ul>

```

Reply:
I created an array of objects to map and display each book.